http.go

```diff
@@ -210,6 +228,18 @@ func httpPutViolationsInner(w http.ResponseWriter, r *http.Request, vs []Violati
 			w.WriteHeader(http.StatusInternalServerError)
 			return
 		}
+
+		// If recovery suppression was specified add the correct timestamp to the reputation
+		// entry. Is suppression is already indicated, only update it if it results in a new
+		// timestamp that is beyond what the existing value is.
```

Comment:
I was thinking we'd just prefer the longer duration and drop the shorter one (the one coming in). This would remove any sort of need for a reputation to be looked up prior to sending a violation/reputation update.

Am up for changing this logic if we think it would make more sense to error out here.

Reply:
Ok yeah, I agree.

Reply:
:+1: will go with what we got, we can change this up later if we need